app/views/admin/attachments/_reference_fields.html.erb

```diff
@@ -3,7 +3,10 @@
   <p class="help-block">This determines the translations of the publication that the attachment will appear in.</p>
   <%= form.select :locale, options_for_locales(attachable.translated_locales), include_blank: 'All languages' %>
 <% end %>
-<%= form.text_field :isbn, label_text: 'ISBN' %>
+
+<%= form.text_field :isbn, label_text: "ISBN (web version)" %>
+<%= form.text_area :meta_data_contact_address, rows: 5, label_text: "Organisation's Contact Details" %>
```

Comment:
Worth checking with the content team about this UI, but it might benefit from a hint about what sort of things to include here.  We can put the address, an email etc... - should probably make that clear.